modules/azure/logic_app_standard/main.tf

```diff
@@ -78,7 +78,7 @@ resource "null_resource" "zip_logic_app" {
   # if check.zip file changes, create deploy.zip file
   provisioner "local-exec" {
     interpreter = local.is_linux ? ["bash", "-c"] : ["PowerShell", "-Command"]
-    command     = local.is_linux ? "cd ${path.module} && mkdir -p files && cd ${var.workflows_source_path} && zip -rq $OLDPWD/files/deploy.zip ." : "New-Item -Path \"${path.module}\" -Name \"files\" -ItemType \"directory\" -Force; Compress-Archive -Path \"${var.workflows_source_path}\\*\" -DestinationPath \"${path.module}\\files\\deploy.zip\""
+    command     = local.is_linux ? "cd ${path.module} && mkdir -p files && cd ${var.workflows_source_path} && zip -rq $OLDPWD/files/deploy.zip ." : "New-Item -Path \"${path.module}\" -Name \"files\" -ItemType \"directory\" -Force; Compress-Archive -Path \"${var.workflows_source_path}\\*\" -DestinationPath \"${path.module}\\files\\deploy.zip\" -Force"
```

Comment:
This can't break stuff for existing uses of this module?

Reply:
Well, I doubt this was ever working on Windows at all (our pipelines run on Linux so it's not an issue there, just locally). Without this param, the apply command runs just for the first time, and every other time, it fails with an error as the file exists already (unless you delete it somehow manually
![image](https://github.com/user-attachments/assets/be989d71-e8c8-46cb-9360-f7ab5ec4b996)